src/pages/studyView/StudyViewPage.tsx

```diff
@@ -191,14 +208,42 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                                             }
                                         }
                                     </Observer>
+                                    {(this.props.routing.location.query.tab === undefined || this.enableAddChartInTabs.includes(this.props.routing.location.query.tab))
+                                    && (
+                                        <div style={{display: 'flex'}}>
+                                            <DefaultTooltip
+                                                trigger={["click"]}
+                                                placement={"bottomLeft"}
+                                                destroyTooltipOnHide={true}
+                                                overlay={() => (
+                                                    <div style={{width: '300px'}}>
+                                                        <CustomCaseSelection
+                                                            selectedSamples={this.store.selectedSamples.result}
+                                                            submitButtonText={"Select"}
+                                                            disableGrouping={true}
+                                                            queriedStudies={this.store.queriedPhysicalStudyIds.result}
+                                                            onSubmit={(chart: NewChart) => {
+                                                                this.store.resetFiltersAndAddCustomChart(chart);
+                                                            }}
+                                                        />
+                                                    </div>
+                                                )}
+                                            >
+                                                <button className='btn btn-primary btn-xs'
 
-                                    {(this.props.routing.location.query.tab === undefined || this.enableAddChartInTabs.includes(this.props.routing.location.query.tab)) &&
-                                    <AddChartButton
-                                        store={this.store}
-                                        currentTab={this.props.routing.location.query.tab ? this.props.routing.location.query.tab : ''}
-                                        addChartOverlayClassName='studyViewAddChartOverlay'
-                                        disableAddGenomicButton={this.props.routing.location.query.tab === StudyViewPageTabKeys.CLINICAL_DATA}
-                                    />}
+                                                        style={{marginLeft: '10px'}}>Custom Selection
+                                                </button>
+                                            </DefaultTooltip>
+                                            <AddChartButton
+                                                key={'addChart'}
```

Comment:
There were two AddChartButton in the previous implementation, thought to use that to differentiate the components, but it would not make much sense since they are not under a for loop. I will remove this.